cumulusci/tasks/package_2gp.py

```diff
@@ -454,6 +461,45 @@ def _create_version_request(
         )
         return response["id"]
 
+    def _resolve_ancestor_id(self, spv_id: str = None) -> str:
+        """
+        If an ancestor_id (04t) is not specified, get it
+        from the latest production release.
+
+        @param spv_id The SubscriberPackageVersionId (04t) that is the ancestor
+        to the version being created.
+        """
+        if not spv_id:
+            tag_name = self.project_config.get_latest_tag(beta=False)
```

Comment:
This will raise GithubException if no release is found. In that case I think we want to catch the exception and use an empty ancestor id (otherwise it won't be possible to create package versions before the first release)